### PR TITLE
Fixed locators in configure/tasks

### PR DESCRIPTION
**Pull request by psav**
Locators for buttons needed updating

{{pytest: -k 'test_permissions and allowed'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f22f4717f6588fb5fcb9931b31686a3d7b9c4bc4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/configure/tasks.py [hunk 1]
```diff
@@ -24,9 +25,12 @@
...
     locators=dict(
-        default="//*[@id='buttons_off']/li[3]/a/img",
-        apply="//*[@id='buttons_on']/li[1]/a/img",
-        reset="//*[@id='buttons_on']/li[2]/a/img"
+        default=pick({LOWEST: "//*[@id='buttons_off']/li[3]/a/img",
+                 '5.4': "//*[@id='buttons_off']/a"}),
+        apply=pick({LOWEST: "//*[@id='buttons_on']/li[1]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[1]"}),
+        reset=pick({LOWEST: "//*[@id='buttons_on']/li[2]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[2]"})
```
Comment (mfalesni) [inline cfme/configure/tasks.py]: Are you sure about the indentation? I personally prefer this style in this case:

```python
reset=pick({
    LOWEST: xyz,
    "5.4": xyz,
})
```

Maybe these buttons can be handled with formbutton since the formbuttons were usually placed in the buttons_on/off divs

Comment (psav) [inline cfme/configure/tasks.py]: hmmm...yeh not sure why it goes screwed up like that.
How does form buttons find a button? By Name?

Comment (mfalesni) [inline cfme/configure/tasks.py]: by `alt`

Comment (psav) [inline cfme/configure/tasks.py]: I'm fine with it, but can we get this merged in now, and do it in another PR? probably good to do a review of all "buttons"

Comment (mfalesni) [inline cfme/configure/tasks.py]: Ok, I will merge it then.
